src/Symfony/Component/DependencyInjection/Tests/Dumper/PreloaderTest.php

```diff
@@ -27,8 +27,11 @@ public function testPreload()
         $preloaded = [];
 
         $r->invokeArgs(null, ['Symfony\Component\DependencyInjection\Tests\Fixtures\Preload\Dummy', &$preloaded]);
-
         self::assertTrue(class_exists('Symfony\Component\DependencyInjection\Tests\Fixtures\Preload\Dummy', false));
+
+        $r->invokeArgs(null, ['Symfony\Component\DependencyInjection\Tests\Fixtures\Preload\DummyWithInterface', &$preloaded]);
+        self::assertFalse(class_exists('Symfony\Component\DependencyInjection\Tests\Fixtures\Preload\DummyWithInterface', false));
+
```

Comment:
I'd create a separate test for this case instead of merging it into the two existing tests.